modules/packages/freebsd_ports

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh -e
+
+command=$1
+
+while read -r line; do
+  export INPUT_$line
+done
+
+list_updates() {
+  pkg version -v -l "<" | sed -Ee $'s/([^ ]+)-[^-]+ .* (.*)\\)/Name=\\1\\\nVersion=\\2\\\nArchitecture=freebsd:10:x86:64/'
```

Comment:
This is the place where the Architecture is faked. We can easily get the arch of each package that's installed from `pkg info` but when we get a list of available updates from `pkg version` that information isn't there.

This could be fixed but it might be tricky, either look up what each package is or combine the two lists somehow.

One thing I want to mention is that I don't think Architecture serves any purpose on this platform – from my understanding all packages installed will be the same arch. I tried installing some Linux packages on FreeBSD (which counts as a different arch in a sense) and they still showed up as freebsd:10:x86:64. Is Architecture strictly necessary? Leaving it out entirely caused issues, can it be blank? Could it be made optional within cfengine core?